src/pages/UserAccountEdit.vue

```diff
@@ -28,7 +28,9 @@ if (!account) {
 }
 
 const registeredServices = computed(() =>
-  me.accounts.map(account => account.type)
+  me.accounts
+    .map(account => account.type)
+    .filter(accountType => accountType !== account.type)
```

Comment:
accountって言うのが現在ページで編集中のアカウントのことであれば、除外するフィルターじゃなくって追加メソッドを実行すべきなんじゃないかと思います
```suggestion
    .push(accountType => accountType !== account.type)
```

Reply:
`registeredServices`に入ってるものは再度選択させたくないのでselectに表示させないようにしているのですが、編集中のアカウントは表示されてほしいので、`registeredServices`には入れないようにしたくて、除外しています

Reply:
あ、`registeredServices`って除外リストか
あ〜、除外リストから除外しているので問題ないということですね